templates/onboarding/setup_done.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+  <div class="flex-fill align-items-center mt-8">
+    <!-- Image container -->
+    <div class="mt-4 mb-4 text-center" style="display:block;">
+      <a class="" href="{{ url_for('dashboard.index') }}">
+        <picture>
+          <source media="(max-width: 650px)" srcset="/static/logo.svg">
+          <img src="/static/logo.svg" style="width: 24rem" alt="logo">
+        </picture>
+      </a>
+    </div>
+
+    <!-- Text container -->
+    <div class="mt-8 mb-4 text-center">
+      <h2 class="text-black-50" style="font-size:2rem">Performing the extension setup...</h2>
+    </div>
+  </div>
+
+  <script type="text/javascript">
+      setTimeout(function() {
+          const data = { tag: "PERFORM_EXTENSION_SETUP" };
+          window.postMessage(data, "/");
+      }, 300);
```

Comment:
Do you know why do we need a timeout here?

Reply:
Oh I see. In that case should we send several times (with delay between each call) the `PERFORM_EXTENSION_SETUP` message to be sure it'll arrive to the extension content script? Currently I see the setup is done for every PERFORM_EXTENSION_SETUP message, maybe we can modify the extension to only run the setup one:
https://github.com/simple-login/browser-extension/blob/13a4836b9d05b045334ddbfd1ed629c82f054e33/src/content_script/input_tools.js#L255

Reply:
I'll try to implement that

Reply:
Changed the  `setTimeout` for `setInterval` and updated the extension code so it only processes the event once.